### PR TITLE
Add Doppel Pack

### PR DESCRIPTION
**Pull request by herbix**
The cards are from my mod time walker.

image

### CODE REVIEW

File: src/main/java/thePackmaster/actions/doppelpack/EmergencyContactAction.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                this::afterSelected,
+                text,
+                ActionType.CARD_MANIPULATION,
+                amount,
+                false
+        ));
+    }
+
+    private void afterSelected(SelectFromGridAction.Source[] sources, AbstractCard[] abstractCards) {
+        if (affectedCards != null) {
```
Comment (erasels) [inline src/main/java/thePackmaster/actions/doppelpack/EmergencyContactAction.java:39]: Under what circumstances can affected cards be null? Should be an empty list at worst whcih works fine in a for-each loop

File: src/main/java/thePackmaster/actions/doppelpack/XCardAction.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
+package thePackmaster.actions.doppelpack;
```
Comment (erasels) [inline src/main/java/thePackmaster/actions/doppelpack/XCardAction.java:1]: Already exists, it's called EasyXCostAction

File: src/main/resources/anniv5Resources/localization/eng/doppelpack/Cardstrings.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    "NAME": "Turbulence",
+    "DESCRIPTION": "Whenever you ${ModID}:Summon a ${ModID}:Doppel, trigger its passive ability{@@}{!M!|1=|2= twice|@= !M! times}."
+  },
+  "${ModID}:Vortex": {
+    "NAME": "Vortex",
+    "DESCRIPTION": "Deal !D! damage. NL If you haven't played this card this turn, ${ModID}:Summon a ${ModID}:Doppel with this card."
+  },
+  "${ModID}:WormHole": {
+    "NAME": "Worm Hole",
+    "DESCRIPTION": "Gain !M! Strength and Dexterity. NL Strength or Dexterity increases damage or Block of ${ModID}:Doppels accordingly."
```
Comment (erasels) [inline src/main/resources/anniv5Resources/localization/eng/doppelpack/Cardstrings.json:49]:
```suggestion
    "DESCRIPTION": "Gain !M! Strength and Dexterity. NL Strength and Dexterity affect ${ModID}:Doppels."
```
Unless I'm missing something, this is more concise.

File: src/main/resources/anniv5Resources/localization/eng/doppelpack/Keywordstrings.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "PROPER_NAME": "Summon",
+    "NAMES": [
+      "summon"
+    ],
+    "DESCRIPTION": "Summon a #yDoppel and put it into the first slot. If you have no empty slots, your first #yDoppel is automatically #yEvoked to make room."
```
Comment (erasels) [inline src/main/resources/anniv5Resources/localization/eng/doppelpack/Keywordstrings.json:7]:
```suggestion
    "DESCRIPTION": "#yChannel a #yDoppel and put it into the first slot. If you have no empty slots, your first #yDoppel is automatically #yEvoked to make room."
```
Might be good to use the channel keyword here to make sure players understand Doppels are basically orbs and occupy the same space.

> put it into the first slot
Should this be the first **free** slot?

**Review by erasels — CHANGES_REQUESTED** [review]

File: src/main/java/thePackmaster/actions/doppelpack/EmergencyContactAction.java (hunk 1 above)
Comment (herbix) [reply src/main/java/thePackmaster/actions/doppelpack/EmergencyContactAction.java]: This is updated now to make sure it's always non-null.

**Comment by herbix** [comment]
@erasels 
I don't have anything want to change. It's ready to merge. Thank you!

**Pull request merged**